user-access-public-model/src/main/java/no/unit/nva/useraccessservice/model/CreateExternalUserRequest.java

```diff
@@ -0,0 +1,29 @@
+package no.unit.nva.useraccessservice.model;
+
+import static nva.commons.core.attempt.Try.attempt;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import no.unit.nva.identityservice.json.JsonConfig;
+
+public class CreateExternalUserRequest {
+
+    private static final String CLIENT_NAME = "clientName";
+
+    @JsonProperty(CLIENT_NAME)
+    private String clientName;
```

Comment:
Would it be a good idea to allow for setting valid redirect urls? How does it work now? Does it default to *?

Reply:
Redirect URLs I belive are for user authentication, and not needed for machine to machine.